sc8280xp.c

```diff
@@ -418,6 +426,7 @@ static struct measure_clk sc8280xp_clocks[] = {
 	{ "disp1_cc_mdss_vsync_clk",				&gcc, 0x82, &disp1_cc, 0x17 },
 	{ "disp1_cc_sleep_clk",					&gcc, 0x82, &disp1_cc, 0x46 },
 	{ "disp1_cc_xo_clk",					&gcc, 0x82, &disp1_cc, 0x45 },
+	{ "measure_only_mccc_clk", 				&gcc, 0xfeedbeef, &mc_cc, 0x50 },
```

Comment:
So far downstream always defined a mux value for `gcc` and the current code even configures this `0xfeedbeef` "mux value" to `gcc` before calling the custom `measure()` function:

https://github.com/andersson/debugcc/blob/1f2d56984ec60e6ca0a18718c75c4e593542cefc/debugcc.c#L214-L217

Are you sure this is correct, since you gave absolutely zero context in the commit message whatsoever?

Reply:
I guess @andersson is indifferent about this too :wink:

Reply:
In other words, I see why it is not required now (because for reading mccc we bypass the gcc's debug mux). But there still should be some more correct value, but without the downstream driver or docs we can not deduce it.

Reply:
@MarijnS95 The patch should be quite easy
[debugcc.txt](https://github.com/andersson/debugcc/files/11365475/debugcc.txt)

However I suppose that we should fix it instead. @andersson do you possibly have a more correct value?

Reply:
@MarijnS95 done, #20 (untested)

Reply:
I never said it would be hard :wink: